### ISSUE TITLE
Update didn't work with redis sysctls

### ISSUE THREAD

**Issue by speendo** [mailcow/mailcow-dockerized#4400]
I just updated mailcow using the updater script in a proxmox container.

It failed with this error
![image](https://user-images.githubusercontent.com/1668429/147364024-d7a28fc8-3706-418e-a690-ed6d6f690c82.png)

After applying the changes @kenji21 in https://github.com/mailcow/mailcow-dockerized/issues/1629#issuecomment-975180177_

it worked. Don't know, if this fix does any harm...

**Comment by ghost** [comment]
Not a bug in Mailcow, you should use specific features flags for your containers (eg. nested).

**Comment by speendo** [comment]
Thanks for your reply. On a holiday! Wow!

Actually I did have the flags I thought would be userful for this container and still got the error. See here:

![image](https://user-images.githubusercontent.com/1668429/147380945-b1037de8-68d6-4869-9abd-a113bf630899.png)

Did I miss an important flag?

**Comment by ghost** [comment]
To be honest I don't know the specific flags but my guess now would be a privileged container. Note that undermine a bit the purpose of a "container" itself.

My personal advice is to go with a KVM virtual machine instead, much more robust and stable for the times coming. Less issue sources, more peace of mind...

**Comment by speendo** [comment]
I appreciate your suggestion and I understand your point.
However, the architecture was a result of a long process and was decided not only by me but also by other people working on the server. We knowingly moved away from KVM, mainly in order to allocate ressources (especially RAM) more efficient. Therefore, the architecture cannot be changed.

However, I wonder if this is even the cause for the issue as the problem occured only after I ran the update-script. Before that, it worked fine with exactly the flags and configuration I showed above.
